pysat/utils.py

```diff
@@ -367,19 +378,22 @@ def load_netcdf4(fnames=None, strict_meta=False, file_format=None,
                         new_index_name = index_name
                     else:
                         # using integer indexing
-                        new_index = np.arange(loop_lim*step_size, dtype=int) % step_size
+                        new_index = np.arange(loop_lim*step_size,
+                                              dtype=int) % step_size
```

Comment:
Need spaces about '*' here and elsewhere

Reply:
fixed